index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>My portfolio</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="./assets/libraries/fontawesome/css/all.css" rel="stylesheet">
+        <link rel="stylesheet" href="./assets/css/style.css">
+    </head>
+    <body>
+        <div class="container">
+            <header class="header">
+                <div class="logo">
+                    My Logo
+                </div>
+                <div class="menu">
+                    <i class="fas fa-bars"></i>
+                </div>
+            </header>
+            <div class="main-content">
+                    <h2 class="title">I'm Richard Glad to see you!</h2>
+                    <p class="intro-text">
+                        I’m a software developer! I can help you build a product, feature or website Look through some of my work and
+                        experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                    </p>
+                    <span class="action-connect">Let's Connect</span>
+                    <div class="social-media-buttons">
+                        <i class="fab fa-twitter"></i>
+                        <i class="fab fa-linkedin-in"></i>
+                        <i class="fab fa-medium-m"></i>
+                        <i class="fab fa-youtube"></i>
+                        <i class="fab fa-pinterest"></i>
+                    </div>
```

Comment:
- [ ] I would suggest wrapping these icons inside the `<ul>` tag (unordered list) because this is the common convention when listing elements. Also, don't forget to use the `<li>` tag (list item) for each element inside the list. [Here you can read and see a few examples about it](https://www.w3schools.com/tags/tag_ul.asp) :hugs:

  ![image](https://user-images.githubusercontent.com/81522435/138315610-ad362f33-79a7-4d81-900a-72f5cb9aad95.png)

- [ ] Also, these images should be links to your social networks. Please use the `<a>` for these images. Each list item should look like: `<li><a href="SOCIAL_URL"><img src="./Images/..." alt="Icon"></a></li>` :memo: